### DISCUSSION TITLE
Mp3tag Portable Configured Out Of The Box Possible?

### DISCUSSION THREAD

**Discussion by bonyboy** [seanap/Plex-Audiobook-Guide#60] [Ideas]
I noticed that [there is a portable version of Mp3tag](https://www.mp3tag.de/en/portable.html) and was wondering if it could be used to have it configured for use out of the box?

**Comment by seanap** [comment]
Not really, everyone's folders are going to be in different locations.  We would need user input to write your specific folder locations into the various scripts and settings.
